test-runner-poc/lib/filters/mongodb_topology_filter.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-
+const MongoClient = require('mongodb').MongoClient;
```

Comment:
this is requiring in a dependency on `mongodb`, which may not be the `mongodb` at `index.js`. You should probably require in:

```js
const MongoClient = require('../../../index`).MongoClient;
```

Reply:
replaced it!